spec/lib/sidekiq_queue_metrics/job_death_middleware_spec.rb

```diff
@@ -0,0 +1,39 @@
+describe Sidekiq::QueueMetrics::JobDeathMiddleware do
```

Comment:
@bilby91 Can we add one positive test case as well where we are calling the JobDeathMonitor.proc. Right now all the tests check that the JobDeathMonitor.proc is not called.

Reply:
I think we should be good now!